packages/usa-banner/src/usa-banner.twig

```diff
@@ -12,32 +12,49 @@
   <div class="usa-accordion">
     <header class="usa-banner__header">
       <div class="usa-banner__inner">
-        <div class="grid-col-auto">
-          <img class="usa-banner__header-flag" src="./img/us_flag_small.png" alt="U.S. flag">
-        </div>
-        <div class="grid-col-fill tablet:grid-col-auto">
+        <img class="usa-banner__header-flag" src="./img/us_flag_small.png" alt="U.S. flag">
+        <div class="usa-banner__header-content">
           <p class="usa-banner__header-text">{{ banner.text }}</p>
-          <p class="usa-banner__header-action" aria-hidden="true">{{ banner.action }}</p>
+          <p class="usa-banner__header-action" aria-hidden="true">
+            {{ banner.action }}
```

Comment:
There appears to be an extra space to the end of the action text that isn't there in the prior version. 
![image](https://user-images.githubusercontent.com/93996430/181789642-31ef5866-815d-40b9-aca3-6e61ba3ebb29.png)

Reply:
Funnily enough, this is caused from breaking the markup into its own line.

```diff
- <p class="usa-banner__header-action" aria-hidden="true">{{ banner.action }}</p>
+ <p class="usa-banner__header-action" aria-hidden="true">
+  {{ banner.action }}
+ </p>
```

Fixed in 4910eb324738f3d1404b79687e5b1d947f38ce80.